src/Distribution/Server/Util/CabalRevisions.hs

```diff
@@ -39,6 +40,9 @@ import Distribution.Version
 import Distribution.Compiler (CompilerFlavor)
 import Distribution.FieldGrammar (prettyFieldGrammar)
 import Distribution.Fields.Pretty (PrettyField (..), showFields)
+#if MIN_VERSION_Cabal(3,7,0)
+import Distribution.Fields.Pretty (pattern NoComment)
+#endif
```

Comment:
I want this file to remain in sync with its copy in `hackage-cli`, so the CPP would stay here.